docs/source/newsfragments/4551.change.1.rst

```diff
@@ -0,0 +1 @@
+:class:`bool(Logic) <cocotb.types.Logic>` casts now treat ``H`` as ``True`` and ``L`` ``False``, respectively, instead of raising :exc:`ValueError`.
```

Comment:
```suggestion
:class:`bool(Logic) <cocotb.types.Logic>` casts now treat ``H`` as ``True`` and ``L`` as ``False``, respectively, instead of raising :exc:`ValueError`.
```